### ISSUE TITLE
Deadline Extension?

### ISSUE THREAD

**Issue by hildeweerts** [teaching-ml/2021#9]
I am currently finalizing our submission for the Teaching ML workshop and I just noticed on the workshop webpage that the submission deadline was extended to June 25. However, this does not seem to be reflected yet on openreview. 

I just wanted to verify whether the deadline is indeed June 25 11:59PM UTC-0?

Many thanks in advance!

**Comment by psteinb** [comment]
Dear @hildeweerts - 

this already happened. @kmkinnaird adapted the website accordingly. I just checked and to my surprise the openreview.net page of our workshop exposes multiple dates for the submission. :thinking: 

I found:
- Paper Submissions from 05 May 2021, 14:00 CEST until 26 Jun 2021, 05:59 CEST and expires 26 Jun 2021, 06:29 CEST [here](https://openreview.net/group?id=ecmlpkdd.org/ECMLPKDD/2021/Workshop/TeachML/Program_Chairs)
- however, the paper submission page itself shows `Submission Start: May 05 2021 12:00PM UTC-0, End: Jun 23 2021 11:59PM UTC-0` [here](https://openreview.net/group?id=ecmlpkdd.org/ECMLPKDD/2021/Workshop/TeachML)

I'll check in with the openreview.net support.

Thanks for reporting.
Peter
